### PR TITLE
Added an argument for parallelism in flink_execute_sql

### PR DESCRIPTION
**Pull request by andrzejdackiewicz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 ### Added
 
+-   Parallelism argument for flink_execute_sql.
```
Comment (grzegorz8) [inline CHANGELOG.md:7]: No need to duplicate `Added` section. Please add it just below `Support Java 8`

File: streaming_jupyter_integrations/magics.py [hunk 2]
```diff
@@ -244,8 +244,12 @@ def __flink_execute_sql_file_internal(self, statements: Iterable[str], display_r
     @cell_magic
     @magic_arguments()
     @argument("--display-row-kind", help="Whether result row kind should be displayed", action="store_true")
+    @argument("--parallelism", "-par", type=int, help="Flink parallelism to use when running the SQL", required=False, default=1)
     def flink_execute_sql(self, line: str, cell: str) -> None:
         args = parse_argstring(self.flink_execute_sql, line)
+        parallelism = args.parallelism
```
Comment (grzegorz8) [inline streaming_jupyter_integrations/magics.py:250]: let's make it shorter:
```self.s_env.set_parallelism(args.parallelism)```

**Review by grzegorz8 — CHANGES_REQUESTED** [review]

File: streaming_jupyter_integrations/magics.py (hunk 2 above)
Comment (andrzejdackiewicz) [reply streaming_jupyter_integrations/magics.py:250]: Done

File: streaming_jupyter_integrations/magics.py [hunk 3]
```diff
@@ -124,7 +124,8 @@ def _flink_connect_local(self, port: int) -> None:
         conf.set_integer("rest.port", port)
         conf.set_integer("parallelism.default", 1)
         self.s_env = StreamExecutionEnvironment(
-            get_gateway().jvm.org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(  # noqa: E501
+            get_gateway().jvm.org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.createLocalEnvironmentWithWebUI(
```
Comment (grzegorz8) [inline streaming_jupyter_integrations/magics.py:127]: revert

Comment (andrzejdackiewicz) [reply streaming_jupyter_integrations/magics.py]: Done!

**Review by grzegorz8 — APPROVED** [review]

File: streaming_jupyter_integrations/magics.py [hunk 4]
```diff
@@ -244,8 +245,12 @@ def __flink_execute_sql_file_internal(self, statements: Iterable[str], display_r
     @cell_magic
     @magic_arguments()
     @argument("--display-row-kind", help="Whether result row kind should be displayed", action="store_true")
+    @argument("--parallelism", "-par", type=int, help="Flink parallelism to use when running the SQL", required=False,
```
Comment (niladrem) [inline streaming_jupyter_integrations/magics.py:248]: please change the short flag to `-p`

**Review by niladrem — APPROVED** [review]

**Pull request merged**